contrib/docker/Dockerfile.arm32v7

```diff
@@ -41,7 +41,7 @@ RUN mkdir /opt/litecoin && cd /opt/litecoin \
     && tar -xzvf litecoin.tar.gz $BD/litecoin-cli --strip-components=1 --exclude=*-qt \
     && rm litecoin.tar.gz
 
-FROM debian:bullseye-slim as builder
+FROM debian:bookworm as builder
```

Comment:
why not use the `slim` here?

Reply:
Currently `arm32v7` is not available for production release. I am experimenting with debian's latest version where I want to remove as many explicit package building (gmp, sqlite, zlib etc.) and installations as possible. 

`bookworm-slim` is a slimmed down version where many packages are removed for smaller production deploy-able version. Using slimmed version makes more sense with `final` and `download` stages. However complete version is better for builder stage where we need more packages to build.